django/contrib/auth/forms.py

```diff
@@ -469,42 +469,67 @@ class AdminPasswordChangeForm(forms.Form):
             attrs={"autocomplete": "new-password", "autofocus": True}
         ),
         strip=False,
+        required=False,  # Can be empty if usable_password is False
+        validators=[],  # Password validation happens in the check_password1() method
         help_text=password_validation.password_validators_help_text_html(),
     )
     password2 = forms.CharField(
         label=_("Password (again)"),
         widget=forms.PasswordInput(attrs={"autocomplete": "new-password"}),
         strip=False,
+        required=False,
         help_text=_("Enter the same password as before, for verification."),
     )
+    usable_password = forms.BooleanField(
+        label=_("Allow login?"),
```

Comment:
I am not sure I like this label. Setting an unusable login is also an option if you use a different authentication backends.

Reply:
@xi What label would you like? Would a different authentication backend allow logins with an unusable login? If so, this would need explanation.

Would "Allow login with password?" be better? Help text then could be "If unchecked, the user will not be able to log in using their password."

Reply:
I like your suggestions :+1: